lnwallet/test_utils.go

```diff
@@ -529,3 +536,211 @@ func ForceStateTransition(chanA, chanB *LightningChannel) error {
 
 	return nil
 }
+
+// MockSigner is a simple implementation of the Signer interface. Each one has
+// a set of private keys in a slice and can sign messages using the appropriate
+// one.
+type MockSigner struct {
```

Comment:
So this actually needs to stay in the `input` package after all. I think I did this before making the new `musession` package. Otherwise the input pacakge can't import this as it makes a dep cycle.

Reply:
Nope, that doesn't work either: the `musession.Manager` struct needs to access the musig2 stuff defined in the `input` package....will mess around with some other dep breakers.